.rspec

```diff
@@ -1 +1,2 @@
+--color --format documentation
```

Comment:
was this something manually added or was this automatic?

Reply:
We added it - after much searching, we found that this was the line responsible for the nice summary of test results in the terminal when you run rspec